app_dart/lib/src/service/config.dart

```diff
@@ -83,6 +83,13 @@ class Config {
     return defaultBranches[slug] ?? kDefaultBranchName;
   }
 
+  // GoB retry options
+  static const RetryOptions gobRetryOptions = RetryOptions(
+    maxAttempts: 4,
+    maxDelay: Duration(seconds: 1),
```

Comment:
Should we actually make this longer? Is this issue due to the commit not sync'ing (which may take longer) or is this just some transient connection type issue?

Reply:
It appears that the commit is not present. The call simply looks for it there. I would try starting with this but if we need more time we can revisit.

Reply:
If its a gob sync issue, that could take up to a minute. Not really sure where in the stack this comes in to know what the delay is. Your call on whats appropriate

Reply:
Ah okay. that is some helpful information. Let me adjust the timing then.

Reply:
Done. removed the max delay in order to wait for a realistic amount of time for the commit to propagate.